iree/modules/check/test/success.mlir

```diff
@@ -14,9 +14,11 @@ func @expect_false() attributes { iree.module.export } {
 }
 
 func @expect_all_true() attributes {iree.module.export} {
-  %dev = hal.ex.shared_device : !hal.device
-  %allocator = hal.device.allocator %dev : !hal.allocator
-  %all_true = hal.buffer_view.const %allocator, "HostLocal|DeviceVisible", "All" : !hal.buffer_view = dense<1> : tensor<2x2xi32>
+  %device = hal.ex.shared_device : !hal.device
```

Comment:
<sub>(just commenting on this file to start a thread)</sub>

Lots of test files are updated here. I hope that wasn't too much trouble. Do you think some of these tests were brittle / checking too many things unrelated to their core purpose, or is that just the nature of changing the syntax for widely used ops?

Reply:
Yeah these are all low level - ideally they'd be flow and above such that the HAL is an implementation detail. The tensorlist/strings modules are also touching the HAL directly for the same reasons.